benches/vob.rs

```diff
@@ -0,0 +1,54 @@
+#![feature(test)]
+#![feature(concat_idents_macro)]
```

Comment:
I don't think this feature is needed?

Reply:
oh yeah, that got included from the benchmark I copied from another project.